src/parser/internal/precedences.rs

```diff
@@ -10,13 +10,13 @@ pub enum Associativity {
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
 pub enum Precedence {
     Lowest,
+    Print,
+    Yield,
+    YieldFrom,
     IncDec,
     KeyOr,
     KeyXor,
     KeyAnd,
-    Print,
-    Yield,
-    YieldFrom,
     Assignment,
```

Comment:
@ryangjchandler no idea why, but this fixes the issue, so 🤷‍♂️, we are removing this later anyways.